database/postgres/utils.js

```diff
@@ -0,0 +1,89 @@
+const faker = require('faker');
+const fs = require('fs');
+const args = require('minimist')(process.argv.slice(2));
+const fastcsv = require('fast-csv');
+const postgresDb = require('./index');
+const contains = require('validator/lib/contains');
+
+module.exports = {
+  createProducts: () => {
+    let products = [];
+    const min = 100
```

Comment:
Looks good! How long is this taking you to generate data? My data structure is similarly complicated and it's taking me a long time to generate my data!